config/config.go

```diff
@@ -58,9 +58,10 @@ type Config struct {
 	ConnectRetries    int32    `json:"connectRetries,omitempty" long:"connect-retries" description:"client connect retries, a negative value means unlimited retries."`
 
 	// Cipher config
-	Cipher   string `json:"cipher,omitempty" long:"cipher" description:"Socks proxy cipher. Dummy (no cipher) will not reduce security because NKN tunnel already has end to end encryption." choice:"dummy" choice:"chacha20-ietf-poly1305" choice:"aes-128-gcm" choice:"aes-256-gcm" default:"chacha20-ietf-poly1305"`
-	Password string `json:"password,omitempty" long:"password" description:"Socks proxy password"`
-	UDP      bool   `json:"udp,omitempty" long:"udp" description:"Support udp proxy"`
+	Cipher      string `json:"cipher,omitempty" long:"cipher" description:"Socks proxy cipher. Dummy (no cipher) will not reduce security because NKN tunnel already has end to end encryption." choice:"dummy" choice:"chacha20-ietf-poly1305" choice:"aes-128-gcm" choice:"aes-256-gcm" default:"chacha20-ietf-poly1305"`
+	Password    string `json:"password,omitempty" long:"password" description:"Socks proxy password"`
+	UDP         bool   `json:"udp,omitempty" long:"udp" description:"Support udp proxy"`
+	UDPIdleTime int32  `json:"udpIdleTime,omitempty" long:"udp-idle-time" description:"UDP connections is to be purged after idle time. 0 is for no purge" default:"0"`
```

Comment:
UDP connections is to be purged after idle time
->
UDP connections will be purged after idle time (in seconds)

Reply:
Yes, modified.